ui/apps/platform/src/Containers/SystemConfig/Form/SystemConfigForm.tsx

```diff
@@ -96,13 +96,12 @@ const SystemConfigForm = ({
                         });
                         setSystemConfig(data);
                         setErrorMessage(null);
+                        setSubmitting(false);
```

Comment:
Changes in this file are unrelated to the redux change. A state change (`setSubmitting(false)`) was happening after the component was destroyed due to `setIsNotEditing()`